src/lib/firebase.js

```diff
@@ -4,15 +4,13 @@ import firebase from 'firebase/app';
 import 'firebase/firestore';
 
 // Initalize Firebase.
-// These details will need to be replaced with the project specific env vars at the start of each new cohort.
 var firebaseConfig = {
-  apiKey: { TCL_X_API_KEY },
-  authDomain: { TCL_X_AUTH_DOMAIN },
-  databaseURL: { TCL_X_DATABASE_URL },
-  projectId: { TCL_X_PROJECT_ID },
-  storageBucket: { TCL_X_STORAGE_BUCKET },
-  messagingSenderId: { TCL_X_SENDER_ID },
-  appId: { TCL_X_APP_ID },
+  apiKey: "AIzaSyAZ9nrYGRvhQvwlM9x10dDVp9HXyKQ9LZw",
```

Comment:
@stacietaylorcima Why do we add the API key here in plaintext? Feels a bit unsafe. Or do we not care for TCL?

Reply:
A great question! We used env variables in the past but figured since there isn't much harm in posting this particular project's API keys publicly, it doesn't matter much. The risk is that someone gets access and floods the DB, but we've got limits on the account that will stop it before it runs up a bill. That being said, if your team would rather use env var's I say go for it! It's an important concept to learn about!